docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1329,20 +1333,29 @@ CREATE TRIGGER t_raster BEFORE UPDATE OR DELETE ON public.image
 
 **GitHub**: Issue [#25575](https://github.com/yugabyte/yugabyte-db/issues/25575)
 
-**Description**: If any of the following PostgreSQL features for version 12 and later are present in the source schema, the import schema step on the target YugabyteDB will fail as YugabyteDB is currently PG11 compatible.
-
-- Multirange [datatypes](https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-BUILTIN).
-- UNIQUE NULLS NOT DISTINCT [clause](https://www.postgresql.org/about/featurematrix/detail/392/) in constraint and index.
-- Aggregate [functions](https://www.postgresql.org/docs/16/functions-aggregate.html#id-1.5.8.27.5.2.4.1.1.1.1) - `any_value`, `range_agg`, `range_intersect_agg`.
-- FETCH FIRST … WITH TIES in select [statement](https://www.postgresql.org/docs/13/sql-select.html#SQL-LIMIT).
-- Regex [functions](https://www.postgresql.org/about/featurematrix/detail/367/) - `regexp_count`, `regexp_instr`, `regexp_like`
-- JSON Constructor [functions](https://www.postgresql.org/about/featurematrix/detail/395/) - `JSON_ARRAY_AGG`, `JSON_ARRAY`,  `JSON_OBJECT`, `JSON_OBJECT_AGG`.
-- JSON query [functions](https://www.postgresql.org/docs/17/functions-json.html#FUNCTIONS-SQLJSON-TABLE) - `JSON_QUERY`, `JSON_VALUE`, `JSON_EXISTS`, `JSON_TABLE`.
-- IS JSON predicate [clause](https://www.postgresql.org/about/featurematrix/detail/396/).
-- Foreign key [references](https://www.postgresql.org/about/featurematrix/detail/319/) to partitioned table.
-- Security invoker [views](https://www.postgresql.org/about/featurematrix/detail/389/).
-- COPY FROM command with WHERE [clause](https://www.postgresql.org/about/featurematrix/detail/330/) and ON_ERROR [option](https://www.postgresql.org/about/featurematrix/detail/433/).
-- Deterministic [attribute](https://www.postgresql.org/docs/12/collation.html#COLLATION-NONDETERMINISTIC) in COLLATION objects.
+**Description**: If any of these PostgreSQL features for version 12 and later are present in the source schema, the import schema step on the target YugabyteDB will fail as YugabyteDB is currently PG11 compatible.
+
+- [JSON Constructor functions](https://www.postgresql.org/about/featurematrix/detail/395/) - `JSON_ARRAY_AGG`, `JSON_ARRAY`, `JSON_OBJECT`, `JSON_OBJECT_AGG`.
+- [JSON query functions](https://www.postgresql.org/docs/17/functions-json.html#FUNCTIONS-SQLJSON-TABLE) - `JSON_QUERY`, `JSON_VALUE`, `JSON_EXISTS`, `JSON_TABLE`.
+- [IS JSON predicate clause](https://www.postgresql.org/about/featurematrix/detail/396/).
+- Any Value [Aggregate function](https://www.postgresql.org/docs/16/functions-aggregate.html#id-1.5.8.27.5.2.4.1.1.1.1) - `any_value`.
+- [COPY FROM command with ON_ERROR](https://www.postgresql.org/about/featurematrix/detail/433/) option.
+- [Non-decimal integer literals](https://www.postgresql.org/about/featurematrix/detail/407/).
+- Non-deterministic collations.
```

Comment:
Add hyperlink for this point with this https://www.postgresql.org/docs/12/collation.html#COLLATION-NONDETERMINISTIC

Reply:
```suggestion
- [Non-deterministic collations](https://www.postgresql.org/docs/12/collation.html#COLLATION-NONDETERMINISTIC).
```